pkg/spec/cluster.go

```diff
@@ -112,6 +123,10 @@ type ClusterSpec struct {
 
 	// etcd cluster TLS configuration
 	TLS *TLSPolicy `json:"TLS,omitempty"`
+
+	// PV represents a Persistent Volume resource.
+	// If defined new pods will use a persistent volume to store etcd data.
```

Comment:
// If defined pods will use a persistent volume to store etcd data.
// When pod/node fails, the replacement pod will try to reuse the existing PV.
// When scale up, ...
// When scale down,

Basically, we probably need to describe the pv claime/cleanup policy a little bit here.

Reply:
Adding a TODO here. this is not supported as it is. We are not very strict about versioning here since etcd operator is not stable yet.

Reply:
> Basically, we probably need to describe the pv claime/cleanup policy a little bit here.

Ok. Perhaps lets wait when the proposal (that talks about these logics) is accepted?

> Adding a TODO here. this is not supported as it is. We are not very strict about versioning here since etcd operator is not stable yet.

That probably the problem on splitting spec and implementation changes. Are you asking to a TODO here and removing in the implementation PR?

Reply:
yea. just a simple TODO in the comment. and remove it in the implementation.